pydirl/files_utils.py

```diff
@@ -42,6 +45,9 @@ def directory_to_zipstream(path):
             if not (os.path.isfile(absPath) or os.path.isdir(absPath)):
                 logger.debug('Skipping unknown element: {}'.format(absPath))
                 continue
+            if exclude and exclude.match(file):
```

Comment:
We can avoid to check the file if we know that won't be included in the stream. I mean we can make as first check the regexp one and then the others.